### DISCUSSION TITLE
Centralised logs

### DISCUSSION THREAD

**Discussion by greatestfen** [Security-Onion-Solutions/securityonion#1760] [Unsupported Versions] [answered]
Hello!
We have placed several sensors throughout the network and would like to consolidate all events on the master, and then send the logs to an external server. According to the documentation, this can be done with syslog-ng, however, there is also a condition that sguild is installed. There is no such service on the master server and no sguild-related directories and I suspect the documentation is from an early SO version. 
Any ideas?
Please assist.

**Comment by weslambert** [comment, marked as answer]
You could achieve this with the [syslog output plugin](https://www.elastic.co/guide/en/logstash/7.x/plugins-outputs-syslog.html) for Logstash.  This was included with our Logstash Docker image as of 2.3.10
